ui/app/templates/components/identity/lookup-input.hbs

```diff
@@ -26,7 +26,7 @@
       <Input class="input" @value={{this.paramValue}} placeholder={{capitalize this.param}} />
     </div>
     <div class="control">
-      <button type="submit" class="button is-hidden">Lookup</button>
+      <Hds::Button @text="Lookup" @color="secondary" class="is-hidden" type="submit" />
```

Comment:
what the heck! this is an _invisible_ button?? 
<img width="743" alt="Screenshot 2023-10-18 at 9 38 18 AM" src="https://github.com/hashicorp/vault/assets/68122737/b948c840-dc87-4462-8219-ae20197f00a9">

Reply:
Yup, so the form can submit. Not sure if we want to just show the button instead?